packages/core/pluggableElementTypes/ViewType.ts

```diff
@@ -18,14 +18,21 @@ export default class ViewType extends PluggableElementBase {
 
   displayTypes: DisplayType[] = []
 
+  // allows for views that are 'extended' from other views to be added as a new view type in the plugin manager
+  // without this, the plugin manager cannot distinguish whether relevant displays have been added to the view type's list
+  // and thus will not display tracks that should be accessible within a view that has been extended from another
+  extendedName?: string
+
   constructor(stuff: {
     name: string
     ReactComponent: ViewReactComponent
     stateModel: IAnyModelType
+    extendedName?: string
   }) {
     super(stuff)
     this.ReactComponent = stuff.ReactComponent
     this.stateModel = stuff.stateModel
+    this.extendedName = stuff.extendedName ? stuff.extendedName : undefined
```

Comment:
probably unnecessary ternary here, can just say this.extendedName=stuff.extendedName and if it's undefined, it's undefined :)

Reply:
that doesn't help if it's some other falsy thing though, like '' or 0

Reply:
probably wouldn't worry about those cases too much, i don't think it'll cause too much problem with the removed ternary:)

Reply:
FWIW, I thought I put this in for a reason, but when I tested it sans-ternary with the MLLV plugin, there were no issues...easy enough to add back in if there's a problem.